mlir/test/mlir-vulkan-runner/vector-deinterleave.mlir

```diff
@@ -0,0 +1,81 @@
+// RUN: mlir-vulkan-runner %s \
+// RUN:  --shared-libs=%vulkan-runtime-wrappers,%mlir_runner_utils \
+// RUN:  --entry-point-result=void --debug-only=dialect-conversion | FileCheck %s
```

Comment:
Is the `--debug-only=dialect-conversion` here intentional? Looks like it's not to me.

Reply:
@angelz913 ☝️

Reply:
Removed in #95820.